doc/mgmt/SONiC_PIM-SSM_MgmtFramework_HLD.md

```diff
@@ -0,0 +1,1343 @@
+IP multicast routing table for VRF: default
+  * -> indicates installed route
+  Source          Group           Input         Output        Uptime
+* 71.0.0.22       233.0.0.1       Vlan100       Vlan200       00:41:54
+                                                Vlan201       00:41:59
+------------------------------------------------------------------------------
+
+sonic# show ip mroute vrf Vrf1
+IP multicast routing table for VRF: Vrf1
+  * -> indicates installed route
+  Source          Group           Input         Output        Uptime
+* 51.0.0.11       233.0.0.1       Vlan300       Vlan301       00:41:59
+* 51.0.0.22       233.0.0.1       Vlan300       Vlan301       00:41:54
+                                                Vlan302       00:41:59
+
+--------------------------------------------------------------------------------
+
+sonic# show ip mroute vrf all
+IP multicast routing table for VRF: default
+  * -> indicates installed route
+  Source          Group           Input         Output        Uptime
+* 71.0.0.11       233.0.0.1       Vlan100       Vlan200       00:41:59
+* 71.0.0.22       233.0.0.1       Vlan100       Vlan200       00:41:54
+                                                Vlan201       00:41:59
+* 71.0.0.11       234.0.0.1       Vlan100       Vlan200       00:41:34
+* 71.0.0.33       234.0.0.1       Vlan100       Vlan200       00:41:31
+                                                Vlan201       00:41:44
+* 71.0.0.22       235.0.0.1       Vlan100       Vlan200       00:41:16
+* 71.0.0.33       235.0.0.1       Vlan100       Vlan200       00:41:14
+
+IP multicast routing table for VRF: Vrf1
+  * -> indicates installed route
+  Source          Group           Input         Output        Uptime
+* 51.0.0.11       233.0.0.1       Vlan300       Vlan301       00:41:59
+* 51.0.0.22       233.0.0.1       Vlan300       Vlan301       00:41:54
+                                                Vlan302       00:41:59
+```
+
+```
+show ip mroute [vrf {<vrf-name> | all}] summary
+===============================================
+
+sonic# show ip mroute summary
+IP multicast routing table summary for VRF: default
+Mroute Type      Installed/Total
+(S, G)           6/6
+
+--------------------------------------------------------------------------------
+
+sonic# show ip mroute vrf Vrf1 summary
+IP multicast routing table summary for VRF: Vrf1
+Mroute Type      Installed/Total
+(S, G)           2/2
+
+--------------------------------------------------------------------------------
+
+sonic# show ip mroute vrf all summary
+IP multicast routing table summary for VRF: default
+Mroute Type      Installed/Total
+(S, G)           6/6
+
+IP multicast routing table summary for VRF: Vrf1
+Mroute Type      Installed/Total
+(S, G)           2/2
+
+```
+
+###### 3.3.2.2.2 PIM-SSM (IPv4) Show commands
+
+List of PIM-SSM (IPv4) show Commands:
+* show ip pim [vrf {<vrf-name> | all}] interface [<intf-name>]

```

Comment:
show ip pim vrf all interface detail, show ip pim interface traffic and show ip pim group-type [<ipv4-nbr-address>]. this commands support are not present